uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/build.js

```diff
@@ -4,9 +4,10 @@
   wrapShim: true,
   modules: [
     {name: 'admin-dashboard'},
+    {name: 'absence-tab'},
     {name: 'my-leave'},
     {name: 'manager-leave'},
-    {name: 'absence-tab'}
+    {name: 'waiting-approval-notification'}
```

Comment:
don't tie the name to a specific status, the notification could change to include other statuses and then the name would not be representative anymore. Call it something like `leave-notification-badge`